src/RadixDlt.NetworkGateway.GatewayApi/Services/NetworkConfigurationProvider.cs

```diff
@@ -77,10 +77,12 @@ public interface INetworkConfigurationProvider : INetworkAddressConfigProvider
 {
     Task Initialize(ICapturedConfigProvider capturedConfigProvider, CancellationToken token);
 
+    int GetNetworkId();
```

Comment:
toolkit library uses byte type, any reason for it to be int then?

Reply:
For Krz's benefit - this network id is a byte - although in Core API it's represented as an int32 because OpenAPI can't do bytes :P. It does have a min:0 / max: 255 on though 👍

Reply:
You're obviously right, fixed.